From 44fb453931c611d4822343debe902cca8e1c8afe Mon Sep 17 00:00:00 2001
From: Scott Kitterman <sklist@kitterman.com>
Date: Tue, 26 May 2020 20:08:16 -0400
Subject: [PATCH] Remove spurious requirement for wheel (#596)

* Remove spurious requirement for wheel

* Put wheel back in pyproject.toml

Co-authored-by: Scott Kitterman <scott@kitterman.com>
Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[james.hilliard1@gmail.com: backport from upstream commit
44fb453931c611d4822343debe902cca8e1c8afe]
---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 9319c1c..d82a7fe 100644
--- a/setup.py
+++ b/setup.py
@@ -35,8 +35,7 @@ except ImportError:
 
 
 requirements = ["six"]
-setup_requirements = ["setuptools",
-                      "wheel"]
+setup_requirements = ["setuptools"]
 test_requirements = ["pytest>=3.2.1,!=3.3.0",
                      "hypothesis>=3.27.0"]
 docs_requirements = ["sphinx>=1.6.5",
-- 
2.25.1

